packages/apps/docs/src/scripts/importReadme.mjs

```diff
@@ -46,14 +46,14 @@ export const createSlug = (str) => {
 };
 
 const getTitle = (pageAST) => {
-  // TODO: flatten all children recursively to prevent issue with
+  // flatten all children recursively to prevent issue with
   // E.g. ## some title with `code`
   const node = pageAST.children[0];
   if (node.type !== 'heading' || node.depth !== 2) {
     throw new Error('first node is not a Heading');
   }
 
-  return node.children[0].value;
+  return node.children.filter((child) => child.type === 'text').join(' ');
```

Comment:
Could you please test with a title like in the comment:

```
## some title with `code`
```

I think the items of `node.children` can be arrays as well, so you could try e.g. `node.children.flat()`